src/shared/modules/sidebar/sidebarDuck.ts

```diff
@@ -51,6 +53,7 @@ type DrawerId =
   | 'about'
   | 'project files'
   | 'settings'
+  | typeof GUIDE_DRAWER_ID
```

Comment:
why the use of `typeof` here?

Reply:
typeof GUIDE_DRAWER_ID evaluates to 'guides', so this is just a way to reuse the variable